src/core-manager/index.js

```diff
@@ -106,12 +103,21 @@ export class CoreManager extends TypedEmitter {
     this.#encryptionKeys = encryptionKeys
     this.#autoDownload = autoDownload
 
-    // Make sure table exists for persisting known cores
-    sqlite.prepare(CREATE_SQL).run()
```

Comment:
hoping removing this in favor of assuming that the db instance that's provided is already migrated is an okay decision.